testing/tests/DevExpress.localization/localization.globalize.widgets.tests.js

```diff
@@ -86,6 +86,28 @@ QUnit.test('DateBox should localize whole date in arabic locale', function(asser
     }
 });
 
+QUnit.test('DateBox should not raise error when digits are not default arabic digits', function(assert) {
```

Comment:
Don't you think we should tests it in Intl too?

Reply:
Done